<!DOCTYPE HTML>
<html>
    
<head>
<title>Lab Files</title>
<link href="css/bootstrap.css" rel="stylesheet" type="text/css" media="all">
<!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
<!-- Custom Theme files -->
<link href="css/style.css" rel="stylesheet" type="text/css" media="all"/>
<!-- Custom Theme files -->
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="keywords" content="" />
<script type="application/x-javascript"> addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false); function hideURLbar(){ window.scrollTo(0,1); } </script>
<!--Google Fonts-->
<link href='http://fonts.googleapis.com/css?family=Roboto:400,500,700' rel='stylesheet' type='text/css'>
<link href='http://fonts.googleapis.com/css?family=Ubuntu+Condensed' rel='stylesheet' type='text/css'>
<!--google fonts-->
<script src="js/jquery-1.11.0.min.js"></script>
<script src="js/bootstrap.min.js"></script>
 <style>
			.title {
				font-size: 36px;
				font-weight: bold;
				margin: 30px 0;
				font-family: 'Ubuntu Condensed', sans-serif;
			
			}
			.second_title
			{
				font-size: 24px;
				font-weight: bold;
				margin:30px 30px 30px 0;
				
			}
			.Context{
				margin:30px  0;
			}
		</style>
</head>
<body>
<!--header-top start here-->
<div class="top-header">
	<div class="container">
		<div class="top-header-main">
			<div class="col-md-8 header-address">
	
			</div>
			<div class="col-md-4 top-social">
	
			</div>
			
		  <div class="clearfix"> </div>
		</div>
	</div>
</div>
<!--header-top end here-->
<!--header start here-->
	<!-- NAVBAR
		================================================== -->
<div class="header w3l">
	<div class="fixed-header">	

		    <div class="navbar-wrapper">
		      <div class="container">
		        <nav class="navbar navbar-inverse navbar-static-top">
		             <div class="navbar-header">
			              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
			                <span class="sr-only">Toggle navigation</span>
			                <span class="icon-bar"></span>
			                <span class="icon-bar"></span>
			                <span class="icon-bar"></span>
			              </button>
			              <div class="logo">
			                   <h1><a class="navbar-brand" href="index.html">Au Lab</a></h1>
			              </div>
			          </div>
		            <div id="navbar" class="navbar-collapse collapse">
		            <nav class="cl-effect-16" id="cl-effect-16">
		              <ul class="nav navbar-nav">
                      <li><a class="active" href="index.html" data-hover="Home">Home</a></li>
                      <li><a href="people.html" data-hover="People">People</a></li>
                      <li><a href="research.html" data-hover="Research">Research</a></li>
                      <li><a href="publications.html" data-hover="Publications">Publications</a></li>
                      <li><a href="software.html" data-hover="Software">Software</a></li>
                      <li><a href="news.html" data-hover="News">News</a></li>
                      <li><a href="contact.html" data-hover="Contact">Contact</a></li>
                      <li><a href="file.html" data-hover="File">File</a></li>
		              </ul>
		            </nav>
		            

		            </div>
		            <div class="clearfix"> </div>
		             </nav>
		          </div>
		           <div class="clearfix"> </div>
		    </div>
	 </div>
</div>
<!--header end here-->
<div class="about">
<div class="container">
<div class="about-main">
<div class="about-top">
<h2>Lab Files</h2>
</div>
<div class="container" style="margin-top: 20px;">
	<div class="row">
			
            <li><a href="#identifier1">Operation manual for identfiy novel gene and isoform based on Pacbio data</a></li>
            <li><a href="#identifier2">MeSMLR-seq followup: Nanopore PromethION</a></li>
            <li><a href="#identifier3">Sequencing data of PromethION</a></li>
            <li><a href="#identifier4">Iso-seq (PacBio), NovaSeq (RNA-seq) Configuration</a></li>
            <li><a href="#identifier5">Quantifying the expression of transcripts by salmon version 1.0.0 and output count matrix by R</a></li>
            

            <div class="row"><hr/></div>
            </br>
            <div id="identifier1"><h2>Operation manual for identfiy novel gene and isoform based on Pacbio data</h2></br>
            
            <h3>Step 1. <a href="https://www.pacb.com/support/software-downloads/">Download SMRT Link software </a></h3></br>
            <h3>Step 2. <a href="http://programs.pacificbiosciences.com/l/1652/2018-03-20/3wrpm1/1652/190668/SMRT_Link_Software_Installation__v5.1.0_.pdf">Official Installation Manual</a></h3></br>
            </br><h4>OR</h4></br>
            
            We can use the following command in Linux:</br>
            
            </br>
            <h4>Download:</h4></br>
            <pre>
            <code>
            wget https://downloads.pacbcloud.com/public/software/installers/smrtlink_5.1.0.26412.zip
            unzip smrtlink_5.1.0.26412.zip #zip password: 9rVkq3HT
            </code>
            </pre>
            </br>
            
            <h4>Installation:</h4></br>
            
            Set your user name. Please use your HPC ID and password. </br>
            <pre>
            <code>
            SMRT_USER=yurwang
            </code>
            </pre>
            </br>
            
            You need to enter your HPC password here:</br>
            <pre>
            <code>
            su $SMRT_USER
            </code>
            </pre>

            </br>
            
            Make a direcoty for smrtlink software:</br>
            <pre>
            <code>
            SMRT_ROOT=/Shared/Au/yuru/smrtlink
            </code>
            </pre>
            </br>
            
            
            Installation, caution on directory.</br>
            <pre>
            <code>
            /Shared/Au/Yijia/smrtlink_5.1.0.26412.run --rootdir $SMRT_ROOT
            </code>
            </pre>

            </br>
            <b>There will be several questions during the installation, please see page 8 of installation manual for more details.</b>
            </br>
            </br>
            You may need to change the the memory and upper limit of processing files of your Linux system.</br>
            <pre>
            <code>
            qlogin -q UI -pe smp 4
            ulimit -u 8192
            </code>
            </pre>
            
            </br>
            You can start smrtlink use:
            <pre>
            <code>
            $SMRT_ROOT/admin/bin/smrtportald-initd start
            </code>
            </pre>
            </br>
            
            <b>You can always save the direcoty for easier access:</b>
            <pre>
            <code>
            vi ~/.bashrc #i for editing, wq! for saving
            export PATH="/Shared/Au/yuru/smrtlink/smrtcmds/bin:$PATH" #add this in the file
            source ~/.bashrc #source the file after quit
            </code>
            </pre>
            </br>
            
            <h3>Step 3. <a href="http://research-pub.gene.com/gmap/src/gmap-gsnap-2018-07-04.tar.gz">Download</a> and <a href="http://research-pub.gene.com/gmap/src/README">Install</a> GMAP software. </h3></br>
            
            </br><h4>OR</h4></br>
            
            We can use the following command in Linux:</br>
            
            </br>
            <pre>
            <code>
            wget http://research-pub.gene.com/gmap/src/gmap-gsnap-2018-07-04.tar.gz
            tar –zxvf gmap-gsnap-2018-07-04.tar.gz
            mkdir gmap-2018-07-04_install
            ./configure /Shared/Au/dingjie/install/gmap-2018-07-04_install
            make
            make check
            make install
            </code>
            </pre>
            
            </br>
            Builds a gmap database for a genome to be used by GMAP or GSNAP:
            <pre>
            <code>
            gmap_build -D /Shared/Au-scratch3/dingjie/zebrafish_raw_data/zebrafish_genome_reference_and_annotation/ gmap_index_GRCz11 -d GRCz11 GRCz11.fa
            </code>
            </pre>
            </br>
            
            <h3>Step 4. <a href="https://github.com/yunhaowang/IsoSeqCon/archive/master.zip">Download</a> and <a href="https://github.com/yunhaowang/IsoSeqCon">Install</a> IsoSeqCon software. </h3></br>
            
            </br><h4>OR</h4></br>
            
            We can use the following command in Linux:</br>
            
            </br>
            <pre>
            <code>
            wget https://github.com/yunhaowang/IsoSeqCon/archive/master.zip
            unzip master.zip
            vi ~/.bashrc
            export PATH="/Shared/Au/dingjie/install/IsoSeqCon-master/bin:$PATH"
            source ~/.bashrc
            
            </code>
            </pre>
            </br>
            
            <h3>Step 5. Convert subreads.bam to ccs.bam</h3>
            </br>
            <pre>
            <code>
            ccs --minPasses=0 --minPredictedAccuracy=0.7 /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/r54148_20180127_071523/2_B01/m54148_180127_152338.subreads.bam cell0_1.ccs.bam
            </code>
            </pre>
            </br>
            
            <h3>Step 6. Classify</h3>
            </br>
            <pre>
            <code>
            pbtranscript classify --detect_chimera_nfl cell0_1.ccs.bam cell0_1.fasta
            </code>
            </pre>
            Then, we can get can get the flnc.fasta and nflnc.fasta files.
            </br>
            
            </br>
            <h3>Step 7. Separate nflnc.fasta file</h3>
            </br>
            <pre>
            <code>
            /Shared/Au/dingjie/install/IsoSeqCon-master/utilities/py_isoseqcon_separate_nflnc_fasta.py -i /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_1/output/cell0_nflnc.fasta -s /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_1/output/ cell0_nflnc_sense.fasta -u /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_1/output/ cell0_nflnc_unknown.fasta
            </code>
            </pre>
            This step gives us 3 files: 1) cell0_flnc.fasta (with strand information); 2) cell0_nflnc_sense.fasta (with strand information); and 3) cell0_nflnc_unknown.fasta (without strand information).
            
            </br>
            
            
            </br><h4>OR</h4></br>
            
            We can use another method here, IsoSeqCon guideline from Yunhao: <a href="https://github.com/yunhaowang/IsoSeqCon">Construct full-length gene isoforms using PacBio Isoform Sequencing (Iso-Seq) data</a>.</br>
            
            
            </br>
            <h3>Step 8. Align the above 3 fasta files in Step 7 to reference genome by GMAP aligner</h3>
            </br>
            (1) Align cell0_1_flnc.fasta:</br>
            <pre>
            <code>
            gmap -D /Shared/Au-scratch3/dingjie/zebrafish_raw_data/zebrafish_genome_reference_and_annotation/gmap_index_GRCz11 -d GRCz11 --max-intronlength-middle 500000 --max-intronlength-ends 500000 -t 20 -z sense_force -f samse -n 0 --split-output cell0_1_flnc cell0_1_flnc.fasta
            </code>
            </pre>
            
            
            </br>
            (2) Align cell0_1_nflnc_sense.fasta:</br>
            <pre>
            <code>
            gmap -D /Shared/Au-scratch3/dingjie/zebrafish_raw_data/zebrafish_genome_reference_and_annotation/gmap_index_GRCz11 -d GRCz11 --max-intronlength-middle 500000 --max-intronlength-ends 500000 -t 20 -z sense_force -f samse -n 0 --split-output cell0_1_nflnc_sense cell0_1_nflnc_sense.fasta
            </code>
            </pre>
            
            </br>
            (3) Align cell0_1_nflnc_unknown.fasta:
            <pre>
            <code>
            gmap -D /Shared/Au-scratch3/dingjie/zebrafish_raw_data/zebrafish_genome_reference_and_annotation/gmap_index_GRCz11 -d GRCz11 --max-intronlength-middle 500000 --max-intronlength-ends 500000 -t 20 -f samse -n 0 --split-output cell0_1_nflnc_unknown cell0_1_nflnc_unknown.fasta
            </code>
            </pre>
            </br>
            
            </br>
            <h3>Step 9. Identify novel gene and isoform by IsoSeqCon (version 0.2)</h3>
            </br>
            <pre>
            <code>
            /Shared/Au/dingjie/install/IsoSeqCon-master/bin/isoseqcon -g /Shared/Au-scratch3/dingjie/zebrafish_raw_data/zebrafish_genome_reference_and_annotation/GRCz11.fa -a /Shared/Au-scratch3/dingjie/zebrafish_raw_data/zebrafish_genome_reference_and_annotation/Danio_rerio.GRCz11.92.chr.gtf -l /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_1/output/cell0_1_nflnc_sense.uniq /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_1/output/cell0_1_nflnc_sense.mult /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_1/output/cell0_1_nflnc_unknown.uniq /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_1/output/cell0_1_nflnc_unknown.mult /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_2/output/cell0_2_nflnc_sense.uniq /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_2/output/cell0_2_nflnc_sense.mult /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_2/output/cell0_2_nflnc_unknown.uniq /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_2/output/cell0_2_nflnc_unknown.mult /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_3/output/cell0_3_nflnc_sense.uniq /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_3/output/cell0_3_nflnc_sense.mult /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_3/output/cell0_3_nflnc_unknown.uniq /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_3/output/cell0_3_nflnc_unknown.mult /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_1/output/cell0_1_flnc.uniq /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_1/output/cell0_1_flnc.mult /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_2/output/cell0_2_flnc.uniq /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_2/output/cell0_2_flnc.mult /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_3/output/cell0_3_flnc.uniq /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/classify_cell0_3/output/cell0_3_flnc.mult -c /Shared/Au-scratch3/dingjie/zebrafish_raw_data/TGS/BJXWZ-201801004C/trim_bam/cell0_novel_isoform/cell0.primer_info.csv --tempdir ./cell0_novel_isoform/ --output_gpd ./cell0_novel_isoform/cell0_without_SR.gpd --output_gtf ./cell0_novel_isoform/cell0_without_SR.gtf --mlt_n_ss 0.0
            </code>
            </pre>
            </br><h4>OR</h4></br>
            
            We can use IDP here: <a href="https://github.com/Dingjie-Wang/Manual-for-running-IDP-pipeline">Manual for running IDP pipeline</a>.</br>

            
            </br>
            
            
            
            
            </div>
            
            <div class="row"><hr/></div>
            <div id="identifier2"><h2>MeSMLR-seq followup: Nanopore PromethION</h2></br>
            <h3>Information from Jackson Lab:</h3>
            </br>
            <h3>Cell lines we have:</h3>
            <br>
           
            <b>There are 9 cell lines from <a href="http://www.internationalgenome.org/human-genome-structural-variation-consortium/">The Human Genome Structural Variation Consortium</a>:</b>
            
            <p>
            The Yoruban trio, NA19238, NA19239 and NA19240; </br>
            the Puerto Rican trio HG00731, HG00732, HG00733; </br>
            the Southern Han Chinese trio, HG00512, HG00513, HG00514. </br>
            <b>NA12878, K562</b>
            </p>
            </br>
            
            <h3>Cost of PromethION, Unit in USD:</h3>
            </br>
            <iframe width="600" height="230" style="border:lpx solid#"ccc" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSXI3tGeD8uhkIhMdXxEQU4bPy-2YQzZ_tZjAQgfvOXq196EzQDWUCTUJUfu_4SNEPlMmLhbO7hk0IH/pubhtml?widget=true&amp;headers=false"></iframe>
            </br>
            
            </br>
            <h3>Nanopore sequencing performance:</h3>
            </br>
            <iframe src="https://docs.google.com/presentation/d/e/2PACX-1vTeqLMiuNihyEyH_nxJDFuTgUZdoaPrTURQzynkh0fJgwrtSFUtdPdAEEGRbC141v26oa6b-T_s0etl/embed?start=false&loop=false&delayms=3000" frameborder="0" width="960" height="569" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
            
            </br>
            
            
            
            
            
            <div class="row"><hr/></div>
            <div id="identifier3"><h2>Sequencing data of PromethION</h2></br>
            
            
            <iframe width="1170" height="400" style="border:lpx solid#"ccc" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vRqz-3w5XgeSXbt4do1r11X0b4LCTCXNdnRBcQIqSK0Lz1LsDJryjGHn0po5CBkXipTKLR7uvZYA1e-/pubhtml?widget=true&amp;headers=false"></iframe></br>
            
            </br><iframe width="1170" height="400" style="border:lpx solid#"ccc" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSsEWj1tESORF-7zZxlHhQF1HwTVRRAdhzKpf-jHYjwVmLoFegwIANS-JCA7S8lQM2za8kKd26R-1mq/pubhtml?widget=true&amp;headers=false"></iframe>
            
            </div>

            <div class="row"><hr/></div>
            <div id="identifier4"><h2>Iso-seq (PacBio), NovaSeq (RNA-seq) Configuration </h2></br>
            
            <style>
            table {
            font-family: arial, sans-serif;
            border-collapse: collapse;
            width: 100%;
            }
            
            td, th {
            border: 1px solid #dddddd;
            text-align: left;
            padding: 8px;
            }
            
            tr:nth-child(even) {
            background-color: #dddddd;
            }
            </style>
            
            <table>
            <h3>Iso-seq (Long read PacBio) quotes</h3></br>
            
            <tr>
            <th>Company/University</th>
            <th>SMRT cells</th>
            <th>Sequencing Costs</th>
            <th>Comments</th>
            </tr>
            
            <tr>
            <td>University of Georgia</td>
            <td>8M</td>
            <td>$500 per library preparation</br>
            $2500 per 8M SMRT cell sequencing</td>
            <td>Accept Total RNA extracts</br>
            Turnaround time  18-25 business days</td>
            </tr>
            
            <tr>
            <td>GENWIZ</td>
            <td>1M</td>
            <td>$1750 per library  preparation</br>
            $1800 per 1M SMRT cell  sequencing</td>
            <td>Accept either Total RNA extracts or Frozen cell pellets.</br>
            Recommended quantities for the fresh frozen cell:</br>
            2 tubes, each containing ≥10⁷ cells</br>
            Turnaround time  18-25 business days</td>
            </tr>
            
            <tr>
            <td>UNIVERSITY of WISCONSIN–MADISON</td>
            <td>1M</td>
            <td>$3,885 for the first sample</br>
            $550 for each additional sample</td>
            <td>Accept Total RNA extracts</br>
            (should be DNAse treated)</td>
            </tr>
            
            <tr>
            <td>Brigham Young University
            <b>(Best offer)</b></td>
            <td>1M and 8M</td>
            <td>For PacBio</br>
            $500 per library preparation</br>
            $2300 per 8M SMRT cell sequencing</br>
            $1800 per 1M SMRT cell sequencing</br>
            </br>
            For RNA-seq (Illumina HiSeq)</br>
            $150 a sample for poly(A) library construction</br>
            Each lane costs $2700</td>
            <td>For PacBio</br>
            600ng of Total RNA, ideally (from each group)</br>
            Turnaround time  1-2 weeks</br>
            </br>
            For RNA-seq</br>
            One lane of 2x125 high output HiSeq data generates approximately 500 million paired end reads (250 million in each direction)</td>
            </tr>
            
            <tr>
            <td>University of Washington</td>
            <td>8M</td>
            <td>$548 per library preparation</br>
            $3,493.99 per 8M SMRT cell sequencing  </td>
            <td>600ng of Total RNA, ideally (from each group)</br>
            Turnaround time 5 - 6 weeks</td>
            </tr>
            
            <tr>
            <td>City of Hope</td>
            <td>1M</td>
            <td>$600 per library preparation</br>
            $1570 per 1M SMRT cell sequencing  </td>
            <td>Turnaround time 3 - 4 weeks</td>
            </tr>
            
            <tr>
            <td>Cold Spring Harbor Laboratory</td>
            <td>8M</td>
            <td>$614 per library preparation
            $1860 per 8M SMRT cell sequencing</td>
            <td>Turnaround time 2 months</td>
            </tr>
            </table>
            
            </br>
            <p>
            <b>General requirements for Total RNA extracts, per library (based on the Iso-seq protocols)</b></br>
            <b>Sample volume: </b>At least 300ng of total RNA in a maximum of 7uL, per library.</br>
            <b>RNA quality: </b>Intact RNA is essential. RIN (RNA integrity number) must be ≥7.0 (ideally ≥8.0).</br>
            <b>Buffer:</b> RNA extracts should be in 10 mM Tris, pH 7.5-8.0. Water is okay if it is nuclease free, but buffer is preferred.
            </p>
            
            </br>
            <table>
            <h3>NovaSeq (short read Illumina) quotes</h3></br>
            
            <tr>
            <th>Company/University</th>
            <th>Sequencing Machine</th>
            <th>Sequencing Costs</th>
            <th>Comments</th>
            </tr>
            
            <tr>
            <td>mrdnalab</td>
            <td>S1 NovaSeq</td>
            <td>$190 per library preparation (total price  with discount $3080)</br>
            $9000 for sequencing 32 libraries(total cost $12,080)</td>
            <td>Suggested Total RNA 500ng per replicate</br>
            (can work with low amount as well)</br>
            RIN >6.5</br>
            Turnaround time  typically average 4 weeks</td>
            </tr>
            
            <tr>
            <td>Duke University</br>
            <b>(Best offer)</b></td>
            <td>S1 NovaSeq</td>
            <td>$188 per library preparation</br>
             $7,400 for sequencing 32 libraries
             (total cost $13,416)</td>
            <td>Suggested Total RNA 500ng per replicate</br>
            (can work with low amount, e.g., 100 ng of Total RNA, as well)</br>
            Turnaround time  typically average 4-5 weeks</br>
            Turnaround time  18-25 business days</td>
            </tr>
            
            <tr>
            <td>University of Kansas Medical Center</td>
            <td>S1 NovaSeq</td>
            <td>$243.6 per library preparation</br>
            $9,273 for sequencing 32 libraries
            (total cost $17,081)</td>
            <td>Standard RNA protocols require a minimum of 500ng of Total RNA. RNA must be DNAse treated to remove contaminating DNA prior to submission.
            Also work with lower input (<100ng) or highly degraded (RIN<5) RNA</td>
            </tr>
            
            <tr>
            <td>University of Utah</td>
            <td>S2 NovaSeq</td>
            <td>$195 per library preparation</br>
            $10,300.00 for sequencing 32 libraries
            (total cost $16,540)</td>
            <td>Turnaround time 3 weeks</td>
            </tr>
            
            <tr>
            <td>Indiana University</td>
            <td>S1 NovaSeq</td>
            <td>$265 per library preparation</br>
            $6,884.58 for sequencing 32 libraries
            (total cost $15,364)</td>
            <td>Turnaround time one month</td>
            </tr>
    
            </table>
            
            
            </div>
            
            <div class="row"><hr/></div>
            </br>
            <div id="identifier5"><h2>Quantifying the expression of transcripts by salmon version 1.0.0 and output count matrix by R</h2></br>
            </br>
            <h3>Input files:</h3></br>
            <ul>
            
            <li>Annotation (Annotation is only for version higher than 1.0 plus versions. OSC installed salmon v.1.0.0. The latest version of salmon is 1.2.0. In this case, we only use version 1.0.0):</li>
            </br>
            <pre>
            <code>
            wget ftp://ftp.ensembl.org/pub/release-99/gtf/homo_sapiens//Homo_sapiens.GRCh38.99.gtf.gz
            </code>
            </pre>
            </br>
            
            <li>Transcriptomes (Ensembl):</li>
            </br>
            <pre>
            <code>
            wget ftp://ftp.ensembl.org/pub/release-99/fasta/homo_sapiens//cdna/Homo_sapiens.GRCh38.cdna.all.fa.gz
            </code>
            </pre>
            </br>
            
            <li>Reads (you can quantify any set of reads, say paired-end reads):
            </li>
            </br>
            <pre>
            <code>
            X5608_GuoD_GBM30-con1_V1N_1_S1_L001_R1_001.fastq.gz</br>
            X5608_GuoD_GBM30-con1_V1N_1_S1_L001_R2_001.fastq.gz
            </code>
            </pre>
            </br>
            </ul>
            
            </br>
            <h3>Running Salmon 1.0.0:</h3></br>
            
            <pre>
            <code>
            module load salmon/1.0.0
            </code>
            </pre>
            </br>
            
            </br>
            <h3>Prepare index for Salmon:</h3></br>
            
            <pre>
            <code>
            salmon index -t Homo_sapiens.GRCh38.cdna.all.fa -i transcripts_index
            </code>
            </pre>
            </br>
            
            </br>
            <h3>Quantifying:</h3></br>
            
            <pre>
            <code>
            salmon quant -i transcripts_index -l A -1 X5608_GuoD_GBM30-con1_V1N_1_S1_L001_R1_001.fastq -2 X5608_GuoD_GBM30-con1_V1N_1_S1_L001_R2_001.fastq  --validateMappings -o /fs/project/PCON0005/data/DLG_RNA-seq/191018-transfer/X5608_transcripts_quant
            
            </code>
            </pre>
            Notice: " -l/libType " means library type; "A" allows salmon automatically infer library type for you; see more detail on <a href="https://salmon.readthedocs.io/en/latest/salmon.html#what-s-this-libtype">salmon official website</a>.
            
            </br>
            </br>
            </br>
            <h3>Output files:</h3></br>
            For each read, salmon will output a folder. The files in that folder are showed below:</br>
            <pre>
            <code>
            aux_info  cmd_info.json  lib_format_counts.json  libParams  logs  quant.sf
            </code>
            </pre>
            </br>
            We need to collect quant.sf file for read counts. This is what quant.sf file looks like. The last column is read counts.
            </br>
            <pre>
            <code>
            Name    Length  EffectiveLength TPM     NumReads
            ENST00000434970.2       9       10.000  0.000000        0.000
            ENST00000415118.1       8       9.000   0.000000        0.000
            ENST00000448914.1       13      14.000  0.000000        0.000
            </code>
            </pre>
            </br>
            Download all the quant.sf files for your reads, put all sf files into a new folder, in this case “quant”.
            Open RStudio to combine all read counts, R code is below:
            <pre>
            <code>
            dir <- "/Users/yuruwang/Desktop/quant" # path of files </br>
            filename <- list.files(dir) # list files in this path </br>
            files <- file.path(dir, filename) # store path for each file </br>
            all(file.exists(files)) # check files exists or not </br>
            lst <- lapply(files, read.table, header=TRUE, stringsAsFactors=FALSE) </br>
            # store files into a list </br>
            
            count <- lapply(lst, "[", c("Name", “NumReads”)) # keep only ID and read counts </br>
            res <- do.call(cbind, count) # because the order of ID are same for each sf file, combine the columns of each file. </br>
            colnames(res)[1] <- “Isoform" </br>
            r <- res[,-grep("\\Name$", colnames(res))] # remove all ID column except the first column(Isoform ID) </br>
            colnames(r)[-1] <- filename </br>
            write.csv(r,file = "all_counts.csv",row.names = FALSE) </br>
            </code>
            </pre>
            </br>
            You can then use DESeq2 or Limma-Voom do differential expression analysis. They are R packages. Please take a look of <a href="http://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#countmat">DESeq2</a> and <a href="https://ucdavis-bioinformatics-training.github.io/2018-June-RNA-Seq-Workshop/thursday/DE.html">Limma-Voom</a> examples.
            </br>
			
		</div>
	</div>
</div></div>
</div>


<!--copy rights start here-->
<div class="copy-rights">
	<div class="container">
		<div class="copy-rights-main">
		
        <li>&copy; <a href=" http://www.augroup.org">Kin Fai Au</a>. All Rights Reserved.</li>
        <li><a href="https://www.osu.edu/">The Ohio State University</a>, OH 43210.</li>

		</div>
	</div>
</div>
<!--copy rights end here-->
</body>
</html>
