<!DOCTYPE HTML>
<html>
	<head>
		<title>News</title>
		<link href="css/bootstrap.css" rel="stylesheet" type="text/css" media="all">
		<!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
		<!-- Custom Theme files -->
		<link href="css/style.css" rel="stylesheet" type="text/css" media="all" />
		<!-- Custom Theme files -->
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
		<meta name="keywords" content="" />
		<script type="application/x-javascript">
			addEventListener("load", function() {
				setTimeout(hideURLbar, 0);
			}, false);

			function hideURLbar() {
				window.scrollTo(0, 1);
			}
		</script>
		<!--Google Fonts-->
		<link href='http://fonts.googleapis.com/css?family=Roboto:400,500,700' rel='stylesheet' type='text/css'>
		<link href='http://fonts.googleapis.com/css?family=Ubuntu+Condensed' rel='stylesheet' type='text/css'>
		<!--google fonts-->
		<script src="js/jquery-1.11.0.min.js"></script>
		<script src="js/bootstrap.min.js"></script>
        <style>
			.title {
				font-size: 36px;
				font-weight: bold;
				margin: 30px 0;
				 color: #F7B400;;
				font-family: 'Ubuntu Condensed', sans-serif;
			
			}
			.second_title
			{
				font-size: 24px;
				font-weight: bold;
				margin:30px 30px 30px 0;
				
			}
			.Context{
				margin:30px  0;
			}
		</style>
	</head>
	<body>
		<!--header-top start here-->
		<div class="top-header">
			<div class="container">
				<div class="top-header-main">
					<div class="col-md-8 header-address">

					</div>
					<div class="col-md-4 top-social">

					</div>

					<div class="clearfix"> </div>
				</div>
			</div>
		</div>
		<!--header-top end here-->
		<!--header start here-->
		<!-- NAVBAR
		================================================== -->
		<div class="header w3l">
			<div class="fixed-header">

				<div class="navbar-wrapper">
					<div class="container">
						<nav class="navbar navbar-inverse navbar-static-top">
							<div class="navbar-header">
								<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar"
								 aria-expanded="false" aria-controls="navbar">
									<span class="sr-only">Toggle navigation</span>
									<span class="icon-bar"></span>
									<span class="icon-bar"></span>
									<span class="icon-bar"></span>
								</button>
								<div class="logo">
									<h1><a class="navbar-brand" href="index.html">Au Lab</a></h1>
								</div>
							</div>
							<div id="navbar" class="navbar-collapse collapse">
								<nav class="cl-effect-16" id="cl-effect-16">
								  <ul class="nav navbar-nav">
                                  <li><a class="active" href="index.html" data-hover="Home">Home</a></li>
                                  <li><a href="people.html" data-hover="People">People</a></li>
                                  <li><a href="research.html" data-hover="Research">Research</a></li>
                                  <li><a href="publications.html" data-hover="Publications">Publications</a></li>
                                  <li><a href="software.html" data-hover="Software">Software</a></li>
                                  <li><a href="news.html" data-hover="News">News</a></li>
                                  <li><a href="contact.html" data-hover="Contact">Contact</a></li>
								  </ul>
								</nav>
								

							</div>
							<div class="clearfix"> </div>
						</nav>
					</div>
					<div class="clearfix"> </div>
				</div>
			</div>
		</div>
		<!--header end here-->
		<!--news start here-->
        <div class="about">
        <div class="container">
        <div class="about-main">
        <div class="about-top">
        <h2>News</h2>
        </div>
		<div class="container">
        
        
        <div class="row">
        <div class="col-lg-10">
        <p class="second_title">Jan 17th, 2020: New Publications.</p>
        <div class="Context">
        <b>Performance difference of graph-based and alignment-based hybrid error correction methods for error-prone long reads.</b><br>
        <b><a>Wang, A.</a></b>, <b><a>Au, K.F.</a></b>
        <br><i>Genome Biology.</i> 2020. [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1885-y">Manuscript</a>]
        </div>
        </div>
        </div>
    
			<div class="row">
				<div class="col-lg-10">
					<p class="second_title">Audrey presents our recent work at the Nanopore Community Meeting 2019.</p>
					<img src="images/dna.jpg" width = 450 />
                    <div class="Context">
						
						Our genome has different levels of organization, and at the nucleosome level, DNA strands wrap around protein cores like beads on a string. Positioning of these nucleosomes and changes in chromatin status play important regulatory roles in gene expression. Our new method, MeSMLR-seq, aims to resolve the long-range dynamics that remain poorly understood at the single-molecule level. Briefly, two examples from the paper in Genome Research are discussed: 1) MeSMLR-seq can resolve differential nucleosome organization principles at the transcription start site of silent versus active genes; and 2) MeSMLR-seq can explore the coupled chromatin status of adjacent genes during transcriptional reprogramming. These methods offer promising utility to study multiple layers of epigenetics simultaneously.[<a href="https://nanoporetech.com/resource-centre/single-molecule-long-read-sequencing-reveals-chromatin-basis-gene-expression-0?ite=17668&ito=1641&itq=acb09df6-2c01-4122-90ec-a5b547959682&itx%5Bidio%5D=624930">Video</a>]
					</div>
				</div>
			</div>
            
            
			<div class="row">
				<div class="col-lg-10">
                <p class="second_title">Jun 14th, 2019: Our latest paper of identifying nucleosome occupancy and chromatin accessibility on single DNA molecule is selected as the cover of the August Issue of Genome Research.</p>
					<img src="images/gr.gif" />
					<div class="Context">

						
						<h3>Single-molecule long-read sequencing reveals the chromatin basis of gene expression. </h3>
                        <br><b>Wang, Y., Wang, A., Liu, Z.</b>, Thurman, A., Powers, L.S., Zou, M., Hefel, A., Li, Y., Zabner, J., <b>Au, K.F.</b><br><i> Genome Res.</i> <b>Cover photo of issue August 2019 </b>. [<a href="https://genome.cshlp.org/content/early/2019/06/14/gr.251116.119.full.pdf+html">Manuscript</a>]
                            <br>
                        <br><td>Combining methyltransferase treatment with nanopore sequencing, Wang et al. developed a new experimental approach “MeSMLR-seq” to map nucleosome occupancy and chromatin accessibility at single long DNA molecules. MeSMLR-seq shows consistent bulk-cell results with existing methods (e.g. MNase-seq and ATAC-seq), but more importantly it brings unprecedented power to capture the single-molecule epigenome. Thus, epigenome heterogeneity can be clearly resolved and subtle but meaningful epigenome differences at single molecules can be discovered. For instance, the MeSMLR-seq data provided the quantitative evidence for the correlation between promoter accessibility and transcription activity. The MeSMLR-seq long reads phased >300 nucleosomes and identified chromatin accessibility of up to 40 adjacent genes simultaneously at single molecules - e.g. the MeSMLR-seq data revealed the coupled chromatin status changes of the adjacent glucose transporter genes responding to the glucose concentration changes. It shows potential to study the association of multiple layers of epigenomes, including base modifications at the same molecules.
                        </td>

					</div>
				</div>
			</div>
			<div class="row">
				<div class="col-lg-10">
					<p class="second_title">Jun 15th, 2018: Shuhua Fu, Anqi Wang and Andrew Thurman are going to make presentations in ISMB 2018, July 7th, Chicago.</p>
					<div class="Context">
                    <b>Topics:</b>
                    <br>IDP-denovo: <i>de novo</i> transcriptome assembly and isoform annotation by hybrid sequencing. <b>(Shuhua Fu)</b>
                    <br>Theoretical analysis of graph-based and alignment-based hybrid error correction methods for error-prone long reads. <b>(Anqi Wang)</b>
                    <br>Gene isoform abundance quantification with Third Generation transcriptome sequencing. <b>(Andrew Thurman)</b>
					</div>
				</div>
			</div>
            
            <div class="row">
            <div class="col-lg-10">
            <p class="second_title">Feb 23rd, 2018: IDP-denovo paper is published.</p>
            <div class="Context">
            <b>IDP-denovo: <i>de novo</i> transcriptome assembly and isoform annotation by hybrid sequencing.</b>
            <br><b>Fu, S.</b>, <b>Ma Y.</b>, Yao, H., Xu, Z., Chen, S., Song, J., <b>Au, K.F.</b>
            <br><i>Bioinformatics</i> 2018. [<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty098/4904264">Manuscript</a>]
            </div>
            </div>
            </div>
			
		</div>
	<!--news end here-->
		
		<!--news letter end here-->
		<!--copy rights start here-->
		<div class="copy-rights">
			<div class="container">
				<div class="copy-rights-main">

                    <li>&copy; <a href=" http://www.augroup.org">Kin Fai Au</a>. All Rights Reserved.</li>
                    <li><a href="https://www.osu.edu/">The Ohio State University</a>, OH 43210.</li>

				</div>
			</div>
		</div>
		<!--copy rights end here-->
	</body>
</html>
