<!DOCTYPE HTML>
<html>
	<head>
		<title>Home</title>
		<link href="css/bootstrap.css" rel="stylesheet" type="text/css" media="all">
		<!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
		<!-- Custom Theme files -->
		<link href="css/style.css" rel="stylesheet" type="text/css" media="all" />
		<!-- Custom Theme files -->
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
		<meta name="keywords" content="" />
		<script type="application/x-javascript">
			addEventListener("load", function() {
				setTimeout(hideURLbar, 0);
			}, false);

			function hideURLbar() {
				window.scrollTo(0, 1);
			}
		</script>
		<!--Google Fonts-->
		<link href='http://fonts.googleapis.com/css?family=Roboto:400,500,700' rel='stylesheet' type='text/css'>
		<link href='http://fonts.googleapis.com/css?family=Ubuntu+Condensed' rel='stylesheet' type='text/css'>
        <!--google fonts-->
        <script src="js/jquery-1.11.0.min.js"></script>
        <script src="js/bootstrap.min.js"></script>
        <style>
        .title {
        font-size: 36px;
        font-weight: bold;
        margin: 30px 0;
        color: #F7B400;;
        font-family: 'Ubuntu Condensed', sans-serif;
        
        }
        .second_title
        {
        font-size: 24px;
        font-weight: bold;
        margin:30px 30px 30px 0;
        
        }
        .Context{
        margin:30px  0;
        }
        </style>
        </head>
        <body>
		<!--header-top start here-->
		<div class="top-header">
			<div class="container">
				<div class="top-header-main">
					<div class="col-md-8 header-address">

					</div>
					<div class="col-md-4 top-social">

					</div>

					<div class="clearfix"> </div>
				</div>
			</div>
		</div>
		<!--header-top end here-->
		<!--header start here-->
		<!-- NAVBAR
		================================================== -->
		<div class="header w3l">
			<div class="fixed-header">

				<div class="navbar-wrapper">
					<div class="container">
						<nav class="navbar navbar-inverse navbar-static-top">
							<div class="navbar-header">
								<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar"
								 aria-expanded="false" aria-controls="navbar">
									<span class="sr-only">Toggle navigation</span>
									<span class="icon-bar"></span>
									<span class="icon-bar"></span>
									<span class="icon-bar"></span>
								</button>
								<div class="logo">
									<h1><a class="navbar-brand" href="index.html">Au Lab</a></h1>
								</div>
							</div>
							<div id="navbar" class="navbar-collapse collapse">
								<nav class="cl-effect-16" id="cl-effect-16">
								  <ul class="nav navbar-nav">
                                  <li><a class="active" href="index.html" data-hover="Home">Home</a></li>
                                  <li><a href="people.html" data-hover="People">People</a></li>
                                  <li><a href="research.html" data-hover="Research">Research</a></li>
                                  <li><a href="publications.html" data-hover="Publications">Publications</a></li>
                                  <li><a href="software.html" data-hover="Software">Software</a></li>
                                  <li><a href="news.html" data-hover="News">News</a></li>
                                  <li><a href="contact.html" data-hover="Contact">Contact</a></li>
								  </ul>
								</nav>
								

							</div>
							<div class="clearfix"> </div>
						</nav>
					</div>
					<div class="clearfix"> </div>
				</div>
			</div>
		</div>
		<!--header end here-->
    <div class="about">
	<div class="container">
        <div class="about-main">
            <div class="about-top">
                <h2>Publications</h2>
            </div>
            
		<div class="row">
        <h3>2020</h3>
			<title>Publication</title>
			 <div id="content">
             <p>
             <br><b>Performance difference of graph-based and alignment-based hybrid error correction methods for error-prone long reads.</b><br>
             <b><a>Wang, A.</a></b>, <b><a>Au, K.F.</a></b>
             <br><i>Genome Biology.</i> 2020. [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1885-y">Manuscript</a>]
             </p>
        <div class="row"><hr></div>
        
        <h3>2019</h3>
             <p>
             <br><b>iASPP mediates p53 selectivity through a modularmechanism fine-tuning DNA recognition</b><br>
             Chen, S., Wu, J., Zhong, S., Li, Y., Zhang P., Ma, J., Ren, J., Tan, Y., <b><a>Wang, Y.</a></b>, <b><a>Au, K.F.</a></b>, Siebold, C., Bond, G.L., Chen, Z., Lu, M., Jones, E.Y. and Lu, X.
             <br><i>Proc. Natl. Acad. Sci. USA</i> 2019 Aug 27. 116 (35), 17470-17479 [<a href="https://www.pnas.org/content/116/35/17470">Manuscript</a>]
             </p>
             <p>
             
             <p>
             <br><b>Single-molecule long-read sequencing reveals the chromatin basis of gene expression.</b><br>
             <b>Wang, Y.*</b>, <b>Wang, A.*</b>, <b>Liu, Z.</b>, <b>Thurman, A.</b>, Powers, L.S., <b>Zou, M.</b>, <b>Zhao, Y.</b>, <b>Hefel, A.</b>, <b>Li, Y.</b>, Zabner, J., <b>Au, K.F.</b>
             <br><i> Genome Research.</i> 2019.  [<a href="https://genome.cshlp.org/content/early/2019/06/14/gr.251116.119.full.pdf+html">Manuscript</a>]
             <br><b>* These authors contributed equally to this work</b>
             </p>
             
             <p>
             <br><b>A comparative evaluation of hybrid error correction methods for error-prone long reads</b><br>
             Fu, S., <b>Wang, A., Au, K.F.</b>
             <br><i> Genome Biology.</i> 2019.  [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1605-z">Manuscript</a>]
             <br>
             </p>
             <div class="row"><hr></div>
             
             <h3>2018</h3>
             <p>
             <br><b>EC coupling structural protein junctophilin-2 encodes a stress-adaptive transcription regulator.</b><br>
             Guo, A., Wang, Y., Chen, B., <b>Wang, Y. </b>,Yuan, J., Zhang, L., Hall, D., Wu, J., Shi, Y., Zhu, Q., Chen, C., Thiel, W.H.,
             Zhan, X., Weiss, R.M, Zhan, F., Musselman, C., Pufall, M., Zhu, W.,<b> Au, K.F. </b>, Hong, J., Anderson, M.E., Grueter, C.E., Song, L.S.
             <br><i> Science.</i> 2018.  [<a href="https://science.sciencemag.org/content/sci/362/6421/eaan3303.full.pdf">Manuscript</a>]
             <br>
             </p>
             
             
             <p>
             <br><b>A Statistical Method for Observing Personal Diploid Methylomes and Transcriptomes with Single-Molecule Real-Time Sequencing.</b><br>
             Suzuki, Y., <b> Wang, Y., Au, K.F., </b> Morishita, S.
             <br><i> Genes.</i> 2018.  [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6162384/pdf/genes-09-00460.pdf">Manuscript</a>]
             <br><b># Co-corresponding author</b>
             </p>
             
             <p>
             <br><b>Single cell expression analysis of primate-specific retroviruses-derived HPAT lincRNAs in viable human blastocysts identifies embryonic cells co-expressing genetic markers of multiple lineages.</b><br>
             Glinsky, G., Durruthy-Durruthy, J., Wossidlo, M., Grow, E.J., <b>Weirather, J.L.</b>,<b> Au, K.F.</b>, Wysocka, J., Sebastiano, V.
             <br><i> Heliyon.</i> 2018.  [<a href="https://www.sciencedirect.com/science/article/pii/S2405844017328153">Manuscript</a>]
             <br>
             </p>
             
             
             <p>
             <br><b>Revealing tumor heterogeneity of breast cancer by utilizing the linkage between somatic and germline mutations.</b><br>
             <b>Zou, M., Jin, R., Au, K.F.#</b>
             <br><i> Briefings in Bioinformatics.</i> 2018. [<a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bby084/5091279">Manuscript</a>]
             <br><b># Corresponding author</b>
             </p>
             
             
             <p><span lang=EN style='mso-ansi-language:EN'><br>
             <b>IDP-denovo: <i>de novo</i> transcriptome assembly and isoform annotation by hybrid sequencing.</b><br>
             <b>Fu, S.</b>, <b>Ma Y.</b>, Yao, H., Xu, Z., Chen, S., Song, J., <b>Au, K.F.#</b> <br>
             <i>Bioinformatics.</i> 2018. [<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty098/4904264">Manuscript</a>]
             <br><b># Corresponding author</b>
             <o:p></o:p></span></p>
             
             <div class="row"><hr></div>
             <h3>2017</h3>
             <p><span lang=EN style='mso-ansi-language:EN'><br>
             <span class=GramE><b>Hybrid Sequencing of Full-Length cDNA Transcripts of Stems
             and Leaves in Dendrobium officinale.</b></span><br>
             He, L., <b>Fu, S. </b>, Xu, Z., Yan, J., Xu, J., Zhou, H., Zhou, J., Chen, X.,
             Li, Y., <b>Au, K.F.<sup>#</sup></b>, Yao, H.<sup>#</sup> <br>
             <i>Genes.</i> 2017. [<a href="http://www.mdpi.com/2073-4425/8/10/257">Manuscript</a>]
             <br>
             <b># Corresponding authors</b> <o:p></o:p></span></p>
             
             
             <p><span lang=EN style='mso-ansi-language:EN'><br>
             <b>CF airway smooth muscle transcriptome reveals a role for PYK2.</b><br>
             Cook, D.P., Adam, R.J., <span class=SpellE>Zarei</span>, K., <span
             class=SpellE><b>Deonovic</b></span><b>, B.</b>, <span class=SpellE>Stroik</span>,
             M.R., <span class=SpellE>Gansemer</span>, N.D., <span class=SpellE>Meyerholz</span>,
             D.K., <b>Au, K.F.</b>, Stoltz, <span class=GramE>D.A</span>. <br>
             <i>JCI Insight.</i> 2017. [<a href="https://insight.jci.org/articles/view/95332">Manuscript</a>]
             <o:p></o:p></span></p>
             
             
             <p><br><b>Comprehensive comparison of Pacific Biosciences and Oxford Nanopore Technologies and their applications to transcriptome analysis [version 1; referees: awaiting peer review].</b>
             <br><b>Weirather, J.L.*</b>, de Cesare, M.*, <b>Wang, Y.*</b>, Piazza, P., Vittorio, S., Wang, X.J., Buck, D., <b>Au, K.F.</b>
             <br><i>F1000Research</i> 2017, 6:100 [<a href="http://dx.doi.org/10.12688/f1000research.10571.1">Manuscript</a>]
             <br><b>* These authors contributed equally to this work</b>
             </p>
             <div class="row"><hr></div>
             <h3>2016</h3>
             <p>
             <br><b>IDP-ASE: haplotyping and quantifying allele-specific expression at the gene and gene isoform level by hybrid sequencing.</b><br>
             <b>Deonovic, B.</b>, <b>Wang, Y.</b>, <b>Weirather, J.</b>, Wang, X.J., <b>Au, K.F.</b>
             <br><i>Nucleic Acids Research.</i> 2016. [<a href="http://nar.oxfordjournals.org/content/early/2016/11/28/nar.gkw1076">Manuscript</a>]
             </p>
             
             <div class = "row"><hr></div>
             <h3>2015</h3>
             <p>
             <br><b>A novel primate-specific noncoding RNA modulates human embryo- and pluripotent stem cell fate.</b><br>
             Durruthy-Durruthy J., Sebastiano V., Wossidlo M., Cepeda D., Cui J., Grow E.J., Davila J., Mall M., Wing W.H., Wysocka J., <b>Au, K.F.</b>, Pera, R.R.
             <br><i>Nature Genetics.</i> 2015. [<a href="http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3449.html">Manuscript</a>]
             </p>
             
             <p>
             <br><b>PacBio Sequencing and Its Applications.</b><br>
             <b>Rhoads, A.</b>, <b>Au, K.F.</b>
             <br><i>Genomics, Proteomics, & Bioinformatics.</i> 2015. [<a href="http://www.sciencedirect.com/science/article/pii/S1672022915001345">Manuscript</a>]
             </p>
             
             <p>
             <br><b>Characterization of fusion genes and the significantly
             expressed fusion isoforms in breast cancer by hybrid
             sequencing.</b><br>
             <b><a>Weirather, J.*</a></b>, Afshar, P.T.*, Clark, T., Tseng, E., Powers, L., Underwood, J.G., Zabner, J., Korlach, J., Wong, W.H., <b><a>Au, K.F.</a></b>
             <br><i>Nucleic Acids Research.</i> 2015 [<a href="http://nar.oxfordjournals.org/content/early/2015/06/03/nar.gkv562.long">Manuscript</a>]
             <br><b>* These authors contributed equally to this work</b><br>
             </p>
             
             
             <p>
             <br><b>Full-length transcriptome sequences and splice variants obtained by a combination of sequencing platforms applied to different root tissues of Salvia miltiorrhiza and tanshinone biosynthesis.</b><br>
             Xu, Z., Peters, R.J., <b><a>Weirather, Jason</a></b>, Luo, H., Liao, B., Zhang, X., Zhu, Y., Ji, A., Zhang, B., Hu, S., <b><a>Au, Kin Fai</a></b>, Song, J., Chen, S.
             <br><i>The Plant Journal.</i> 2015 Apr 24 [<a href="http://onlinelibrary.wiley.com/doi/10.1111/tpj.12865/abstract">Manuscript</a>]
             </p>
             
             <p>
             <br><b>Accurate mapping of RNA-Seq data.</b><br>
             <b><a>Au, Kin Fai</a></b>
             <br><i>Methods in Molecular Biology </i> 2015;1269:147-61. [<a href="https://dx.doi.org/10.1007/978-1-4939-2291-8_9">Manuscript</a>]
             </p>
             <div class = "row"><hr></div>
             <h3>2014</h3>
             <p>
             <br><b>The transcriptome of human pluripotent stem cells.</b><br>
             <b><a>Au, Kin Fai*</a></b>, Sebasiano, V.
             <br><i>Current Opinion in Genetics and Development.</i> 2014 Oct 31. 28, 71-77 [<a href="http://www.sciencedirect.com/science/article/pii/S0959437X14001075">Manuscript</a>]
             <br>* <b>Corresponding Author</b>
             </p>
             
             <p>
             <br><b>Human tRNA synthetase catalytic nulls with diverse functions.</b><br>
             Lo, W.S., Gardiner E., Xu, Z., Lau C.F., Wang, F., Zhou, J.J., Mendlein, J.D., Nangle, L.A., Chiang, K.P., Yang, X.L., <b><a>Au, Kin Fai</a></b>, Wong, W.H., Guo, M., Zhang, M., Schimmel, P.
             <br><i>Science.</i> 2014 Jul 18. 345 (6194), 328-32 [<a href="http://www.sciencemag.org/content/345/6194/328.long">Manuscript</a>]
             </p>
             
             <p>
             <br><b>ALS-associated mutation FUS-R521C causes DNA damage and RNA splicing defects</b><br>
             Qiu H., Lee S., Shang Y., Wang W.Y., <b><a>Au, Kin Fai</a></b>, Kamiya S., Barmada S.J., Finkbeiner S., Lui H., Carlton C.E., Tang A.A., Oldham M.C., Wang H., Shorter J., Filiano A.J., Roberson E.D., Tourtellotte W.G., Chen B., Tsai L.H., Huang E.J.
             <br><i>The Journal of clinical investigation.</i> 2014 Feb 10. 124 (3), 0-0 [<a href="http://www.jci.org/articles/view/72723">Manuscript</a>]
             </p>
             
             <div class = "row"><hr></div>
             <h3>2013</h3>
             
             <p>
             <br><b>Oct4-Sall4-Nanog network controls developmental progression in the preimplantation mouse embryo</b><br>
             Tan, M.H.*, <b><a>Au, Kin Fai*</a></b>, Leong, D.E., Foygel. K., Wong, W.H., Yao, M.W.M.
             <br><i>Molecular Systems Biology</i>. 2013. 9:632. [<a href="http://msb.embopress.org/content/9/1/632.long">Manuscript</a>]</br>
             <b>* These authors contributed equally to this work</b>
             </p>
             
             <p>
             <br><b>Characterization of the human ESC transcriptome by hybrid sequencing</b><br>
             <b><a>Au, Kin Fai</a></b>, Sebastiano V., Afshar P.T., Durruthy J.D., Lee L., Williams B.A., Bakel H.V., Schadt E., Pera R.A.R., Underwood J., Wong W.H.
             <br><i>Proc. Natl. Acad. Sci. USA</i> 2013 110 (50) E4821-E4830 [<a href="http://www.pnas.org/content/110/50/E4821.long">Manuscript</a>]
             <br>
             <i></i>
             
             
             <div class = "row"><hr></div>
             <h3>2012</h3>
             <p>
             <br><b>Improving PacBio long read accuracy by short read alignment.</b><br>
             <b><a>Au, Kin Fai</a></b>, Underwood, J., Lee, L., Wong, W.H.
             <br><i>PLoS ONE</i> 2012. 7(10): e46679. doi:10.1371/journal.pone.0046679 [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3464235/">Manuscript</a>] </br>
             
             </p>
             
             <p>
             <br><b>RNA sequencing reveals diverse and dynamic repertoire of the <i>Xenopus tropicalis</i> transcriptome over development. </b><br>
             Tan, M.H.*, <b><a>Au, Kin Fai*</a></b>, Yablonovitch, A.L.*, Wills, A.E., Baker J.C.,
             Wong, W.H., Li, J.B.
             <br><i>Genome Research</i>, 2012. doi:10.1101/gr.141424.112 [<a href="http://genome.cshlp.org/content/23/1/201.long">Manuscript</a>]
             <br><b>* These authors contributed equally to this work</b><br>
             </p>
             
             <p>
             <br><b>Activation of Innate Immunity is Required for Efficient Nuclear Reprogramming</b><br>
             Lee, J., Sayed, N., Hunter, A., <b><a>Au, Kin Fai</a></b>, Wong, W.H., Mocarski, E., Pera, R.R., Yakubov, E., Cooke, J.P.
             <br><i>Cell</i>. 2012 Oct 26.  151 (3):547-58 [<a href="http://www.sciencedirect.com/science/article/pii/S0092867412011816">Manuscript</a>]</br>
             </p>
             
             <div class = "row"><hr></div>
             <h3>2010</h3>
             
             <p>
             <br><b>Detection of splice junctions from paired-end RNA-seq data by SpliceMap </b><br>
             <b><a>Au, Kin Fai</a></b>, Jiang, H., Lin, L., Xing, Y., Wong, W.H.
             <br><i>Nucleic Acids Research</i>. Aug;38(14):4570-8. 2010. [<a href="http://nar.oxfordjournals.org/content/38/14/4570.long">Manuscript</a>]</br>
             </p>
             
             <div class = "row"><hr></div>
             <h3>2008</h3>
             
             <p>
             <br><b>Structures of an alanine racemase from Bacillus anthracis (BA0252) in the presence and absence of (R)-1-aminoethylphosphonic acid (L-Ala-P)</b><br>
             <b><a>Au, Kin Fai</a></b>, Ren, J., Walter, T.S., Harlos, K., Nettleshipa, J.E., Owens, R.J., Stuart, D.I., Esnouf, R.M.
             <br><i>Acta Crystallogr Sect F Struct Biol Cryst Commun</i>. 2008 May 1;64(Pt 5):327-33.  Aug;[<a href="http://scripts.iucr.org/cgi-bin/paper?S1744309108007252">Manuscript</a>]</br>
             </p>
             
             <div class = "row"><hr></div>
             <h3>2006</h3>
             
             <p>
             <br><b>Application of high-throughput technologies to a structural proteomics-type analysis of <i>Bacillus anthracis </i></b><br>
             <b><a>Au, Kin Fai</a></b>, Berrow, N.S., Blagova, E., Boucher, I.W., Boyle, M.P., Brannigan, J.A., Carter, L.G., Dierks, T., Folkers, G.,
             Grenha, R., Harlos, K., Kaptein, R., Kalliomaa, A.K., Levdikov, V.M., Meier, C., Milioti, N., Moroz, O., Muller, A., Owens, R.J., Rzechorzek, N., Sainsbury, S.,
             Stuart, D.I., Walter, T.S., Waterman, D.G., Wilkinson, A.J., Wilson, K.S., Zaccai, N., Esnouf, R.M., Fogg, M.J.
             <br><i>Acta Crystallogr D Biol Crystallogr</i>. 2006 Oct;62(Pt 10):1267-75.[<a href="http://scripts.iucr.org/cgi-bin/paper?S0907444906033555">Manuscript</a>]</br>
             </p>
             
             <p>
             <br><b>Lysine methylation as a routine rescue strategy for protein crystallization</b><br>
             Walter, T.S., Meier, C., Assenberg, R., <b><a>Au, Kin Fai</a></b>, Ren, J., Vermaa, A., Nettleshipa, J.E., Owens, R.J., Stuart, D.I., Grimes, J.M..
             <br><i>Structure</i>. 2006 Nov;14(11):1617-22.[<a href="http://www.sciencedirect.com/science/article/pii/S0969212606003911">Manuscript</a>]</br>
             </p>
             
			 </div>


		</div>

		</div>
	</div>
		<!--copy rights start here-->
		<div class="copy-rights">
			<div class="container">
				<div class="copy-rights-main">
					
                    <li>&copy; <a href=" http://www.augroup.org">Kin Fai Au</a>. All Rights Reserved.</li>
                    <li><a href="https://www.osu.edu/">The Ohio State University</a>, OH 43210.</li>

				</div>
			</div>
		</div>
		<!--copy rights end here-->
	</body>
</html>
